satpy/readers/seviri_l1b_native.py

```diff
@@ -194,17 +196,28 @@ def __init__(self, filename, filename_info, filetype_info,
         self._read_trailer()
         self.image_boundaries = ImageBoundaries(self.header, self.trailer, self.mda)
 
+        self.tres = REPEAT_CYCLE_DURATION  # base RC duration of 15
+        if filetype_info is not None:  # to avoid error in the pytest
```

Comment:
Is it possible to remove that line and instead update the tests to pass a proper `filetype_info`?